### PR TITLE
Document how to build and deploy the snap

### PR DESCRIPTION
**Pull request by samuelallan72**

### CODE REVIEW

**Comment by samuelallan72** [comment]
Revisiting this - I think it would be good to get this into the contributing guide at least.  We can move it later if we want, but there is value in this being documented somewhere at least. :)

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -92,3 +92,16 @@ pip install -e .
 # run cou!
 cou --version
 ```
+
+## Testing the snap
+
+Installing a local snap has some manual steps because it must be installed in 'dangerous' mode:
```
Comment (chanchiwai-ray) [inline CONTRIBUTING.md:98]: I think you can install with devmode, or not?

Comment (samuelallan72) [reply CONTRIBUTING.md:98]: You can install with devmode, but it's safer to install with `--dangerous`.

- `--dangerous`: installs the snap even if signatures from the snap store are missing
- `--devmode`: disables security sandbox, implicitly adds `--dangerous` too.

Comment (chanchiwai-ray) [reply CONTRIBUTING.md:98]: So maybe this is explanation why the developer should test with `dangerous` mode? I feel like "must" mean they don't have other choices, but here I think we are recommending them to test with `dangerous` mode

Comment (samuelallan72) [reply CONTRIBUTING.md:98]: Yeah maybe the wording could be improved - what I mean there is that it cannot be installed normally - you must add the `--dangerous` flag (or of course `--devmode`).

Comment (chanchiwai-ray) [reply CONTRIBUTING.md]: Thanks

**Review by chanchiwai-ray — APPROVED** [review]

**Review by jneo8 — APPROVED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (samuelallan72) [reply CONTRIBUTING.md]: Thanks for your review :)

**Pull request merged**